content/root/quick-start-guide.textile

```diff
@@ -87,70 +108,104 @@ blang[java].
   import io.ably.lib.realtime.*;
   ```
 
 blang[objc].
-  The Ably Realtime and REST library is available as a CocoaPod. Add this line to your application's Podfile:
+  The Ably library is available as a CocoaPod.
+
+  To get started with your project, add the library to your Podfile:
 
-  bc[text]. pod 'Ably'
+  ```[text]
+  pod 'Ably'
+  ```
 
-  And install with @pod install@. Then in your files:
+  Install the library with:
+  
+  ```[sh]
+  pod install
+  ``` 
+  
+  Import the library into your files with:
 
   ```[objc]
   #import "Ably.h"
   ```
 
 blang[swift].
-  The Ably Realtime and REST library is available as a CocoaPod. Add this line to your application's Podfile:
+  The Ably library is available as a CocoaPod.
 
-  bc[text]. pod 'Ably'
+  To get started with your project, add the library to your Podfile:
 
-  And install with @pod install@. Then in your files:
+  ```[text]
+  pod 'Ably'
+  ```
+
+  Install the library with:
+  
+  ```[sh]
+  pod install
+  ``` 
+  
+  Import the library into your files with:
 
   ```[swift]
   import Ably
   ```
 
 blang[csharp].
-  The Ably Realtime and REST client library is available as a "Nuget package":https://www.nuget.org/packages/ably.io/. You can install it from the Package Manager Console using this command:
+  The Ably library is available as a "Nuget package":https://www.nuget.org/packages/ably.io/. 
+  
+  To get started with your project, install the library from the Package Manager Console:
 
-  bc[text]. PM> Install-Package ably.io
+  ```[text]
+  PM> Install-Package ably.io
+  ```
 
 blang[flutter].
-  The Ably library is "hosted on Github":https://github.com/ably/ably-flutter and is available as a "Flutter plugin":https://pub.dev/packages/ably_flutter. Update your @pubspec.yaml@ with appropriate package version:
+  The Ably library is hosted on "GitHub":https://github.com/ably/ably-flutter and available as a "Flutter plugin":https://pub.dev/packages/ably_flutter. 
+  
+  To get started with your project, update your @pubspec.yaml@ with the package and version:
 
-  bc[yaml]. dependencies:
+  ```[yaml]
+  dependencies:
     # ...
     ably-flutter: [version]
     # ...
+  ```
 
-  Now, import the package into your Dart file.
+  Import the package into your Dart file:
+
+  ```[flutter]
+  import 'package:ably_flutter/ably_flutter.dart' as ably;
+  ```
 
-  bc[flutter]. import 'package:ably_flutter/ably_flutter.dart' as ably;
+h2(#step-2). Step 2: Connect to Ably 
 
-h2(#receiving-messages). The basics: Receiving messages
+blang[default].
+  Clients need to authenticate with Ably to establish a realtime connection. The code block below uses a demo API key to authenticate and print the message @Connected to Ably!@ when you've successfully connected.
+
+  Alternatively, you can use your own API key if you already have an account, or "sign up":https://ably.com/signup for a free account to obtain one. Just make sure the key provides the subscribe and publish capabilities. 
```

Comment:
> Just make sure the key provides the subscribe and publish capabilities.

Ensure the API key provides the subscribe and publish capabilities.

Ideally, can we link to where we explain this in the docs. Possibly [here](https://ably.com/documentation/core-features/authentication#capabilities-explained). It looks like we need a topic on API keys in the docs though, as mostly we currently link to the knowledge base.

Reply:
Added that link in. I was trying to avoid too many links out from a quickstart guide, but agree that "capabilities" is a vague name to understand.